src/imu/mpu9255_data.hpp

```diff
@@ -0,0 +1,16 @@
+#ifndef SRC_MPU9255_REGISTERS_HPP_
+#define SRC_MPU9255_REGISTERS_HPP_
+
+#include <cstdint>
+
+namespace imu {
+
+static constexpr std::uint8_t GYRO_CONFIG = 0x1B;
+static constexpr std::uint8_t GYRO_XOUT_H = 0x43;
```

Comment:
This is the original name of the register from the register map of IMU. I could also call it "gyroscope measurements" or so, but I thought it would be easier to find it in the register map?!

Reply:
Ok yeah if this is a name of a register, then it is fine